docs/book/src/topics/ignition-support.md

```diff
@@ -23,17 +23,57 @@ For more generic information, see [Cluster API documentation on Ignition Bootstr
 
 ## Overview
 
-By default machine controller stores EC2 instance user data using SSM to store it encrypted, which underneath
-use multi part mime types, which are [unlikely to be supported](https://github.com/coreos/ignition/issues/1072)
-by Ignition.
+When using CloudInit for bootstrapping, by default the awsmachine controller stores EC2 instance user data using SSM to store it encrypted, which underneath uses multi part mime types.
+Unfortunately multi part mime types are [not supported](https://github.com/coreos/ignition/issues/1072) by Ignition. Moreover EC2 instance user data storage is also limited to 64 KB, which might not always be enough to provision Kubernetes controlplane because of the size of required certificates and configuration files.
 
-EC2 user data is also limited to 64 KB, which is often not enough to provision Kubernetes controlplane because
```

Comment:
This statement still holds good, isn't it?

Reply:
It does and in fact it is still there I moved it in the paragraph above 🙂